src/WebJobs.Script/Description/ScriptFunctionInvokerBase.cs

```diff
@@ -79,6 +79,14 @@ protected override Task InvokeCore(object[] parameters, FunctionInvocationContex
                             Value = stream
                         };
                         await inputBinding.BindAsync(bindingContext);
+
+                        if (bindingContext.Value is string)
```

Comment:
I don't believe it is. @mathewc has the most context here, so I'd like to have his input.